apps/src/templates/sectionAssessments/FreeResponsesSurveyTable.story.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import FreeResponsesSurveyTable from './FreeResponsesSurveyTable';
+import { surveyOne, surveyTwo } from './assessmentsTestHelpers';
+
+export default storybook => {
+  return storybook
+    .storiesOf('SectionAssessments/FreeResponsesSurveyTable', module)
+    .addStoryTable([
+      {
+        name: 'Free responses for survey 1',
+        description: 'Display anonymous responses',
+        story: () => (
+          <FreeResponsesSurveyTable
+            freeResponses={surveyOne}
+          />
+        )
+      },
+      {
+        name: 'Free responses for survey 2',
+        description: 'Table not displayed if less than 5 surveys are submitted',
```

Comment:
oooh, this could be a good case to try out Enzyme tests for this table. Not sure about your timeline or if you've tackled those, but let me know if you want to walk through one together.

Reply:
Agree!  My goal is to write this test after I have hooked up data and I will gladly pair on writing test.  Thanks